### PR TITLE
Usagov 2117 phpcs local

### PR DESCRIPTION
**Pull request by omerida**
## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-2117

## Description

Adds PHPCodeSniffer and the Parallel PHP linter as dev tools for local environments.
Adds composer scripts to run the tools in the composer container.

## Type of Changes

- [ ] New Feature
- [ ] Bugfix
- [ ] Frontend (Twig, Sass, JS)
  - Add screenshot showing what it should look like
- [ ] Drupal Config (requires "drush cim")
- [ ] New Modules (requires rebuild)
- [ ] Documentation
- [ ] Infrastructure
  - [ ] CMS
  - [ ] WAF
  - [ ] WWW
  - [ ] Egress
  - [ ] Tools
  - [ ] Cron
- [x] Other

### Change Requirements

- [ ] Requires New Documentation (Link: {})
- [ ] Requires New Config
- [ ] Requires New Content

### Validation Steps

- run `composer install` to get the dependencies.
- run `./bin/composer phpcs-errors`. You should see a report from phpcs showing no errors
- run `./bin/composer php-lint`. You should see a report from phpcs showing no errors
- run `./bin/composer phpcs-errors-strict`. You should see a report from phpcs showing warnings across a lot of files

There are two commands that tell phpcs to only check files changed in the current branch, `phpcs-changes` and `phpcs-changes-strict`. If the others run, this one should run too. To confirm they work, you'd need to commit a change to some file in the project and run them. If you fix an existing code style issue, you shouldn't see it in the output. If you intentionally create one, it should be added to the report. You could make a commit and just not push any changes back up to here to be safe.
- Test instruction 3

## Security Review

- [ ] Adds/updates software (including a library or Drupal module)
- [ ] Communication with external service
- [ ] Changes permissions or workflow
- [ ] Requires SSPP updates

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.

### CODE REVIEW

**Comment by omerida** [comment]
update to run on drupal 10.3 dev

File: README.md [hunk 1]
```diff
@@ -320,6 +320,24 @@ If you make any changes to the `scss` or `js` files, make sure to check for lint
 
 [back to top](#usagov-2021)
 
+## Checking PHP Code style and syntax errors
+
+PHPCodesniffer and the parallel linting tools should be installed automatically on a local environment via `composer install`. PHPCodeSniffer is used to ensure new code follows Drupal's coding standard. The parallel linter will check for PHP syntax errors. If they detect any errors, they must be fixed before a PR of changes can be accepted.
+
+The following composer scripts are aliases for running these tools. development setup
```
Comment (akf) [inline README.md]: Remove words "development setup"

**Review by akf — DISMISSED** [review]
The only issue I caught is a couple of stray words in the README.md. So I'm approving this, but not merging immediately; if you can get to that on Monday, great! Otherwise I hate to hold it up longer for that.

File: README.md (hunk 1 above)
Comment (omerida) [reply README.md]: done

**Review by akf — APPROVED** [review]

**Pull request merged**